Sources/Build/BuildPlan.swift

```diff
@@ -312,7 +330,13 @@ public final class ClangTargetBuildDescription {
             else if let generatedModuleMapType = clangTarget.moduleMapType.generatedModuleMapType {
                 let path = tempsPath.appending(component: moduleMapFilename)
                 let moduleMapGenerator = ModuleMapGenerator(targetName: clangTarget.name, moduleName: clangTarget.c99name, publicHeadersDir: clangTarget.includeDir, fileSystem: fileSystem)
-                try moduleMapGenerator.generateModuleMap(type: generatedModuleMapType, at: path)
+                let swiftHeaderPath = generateSwiftHeaderInModuleMap
+                    ? tempsPath.appending(component: "\(target.name)-Swift.h") : nil
+                try moduleMapGenerator.generateModuleMap(
+                    type: generatedModuleMapType,
+                    at: path,
+                    swiftHeaderPath:swiftHeaderPath
```

Comment:
Why does `swiftHeaderPath` need to be set now and wasn't before?

Reply:
This is all to generate a module map for a _Mixed_Target. Based on how CocoaPods does it for mixed pods, I suspect it may need to look something like this for a `MixedTarget`:
```
module MixedPackage {
    umbrella header "/Users/nickcooke/Developer/MixedPackage/Sources/MixedPackage/Public/MixedPackage.h"
    export *
}
module MixedPackage.Swift { 
    header "/Users/nickcooke/Developer/MyPackage/.build/x86_64-apple-macosx/debug/MixedPackage.build/MixedPackage-Swift.h"
    requires objc
}
```